flink-table/flink-table-planner/src/main/java/org/apache/flink/table/sqlexec/SqlToOperationConverter.java

```diff
@@ -415,6 +431,59 @@ private Operation convertAlterDatabase(SqlAlterDatabase sqlAlterDatabase) {
 		return new AlterDatabaseOperation(catalogName, databaseName, catalogDatabase);
 	}
 
+	/** Convert CREATE VIEW statement. */
+	private Operation convertCreateView(SqlCreateView sqlCreateView) {
+		final SqlNode query = sqlCreateView.getQuery();
+		final SqlNodeList fieldList = sqlCreateView.getFieldList();
+
+		SqlNode validateQuery = flinkPlanner.validate(query);
+		PlannerQueryOperation operation = toQueryOperation(flinkPlanner, validateQuery);
+		TableSchema schema = operation.getTableSchema();
+
+		if (!fieldList.getList().isEmpty()) {
```

Comment:
ditto

Reply:
ditto, fieldList is optional in view definition.